src/LabelsProviderInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator;
+
+/**
+ * Provides data attribute labels.
+ */
+interface LabelsProviderInterface
+{
+    /**
+     * @return array<string, string> A set of attribute labels.
+     */
+    public function getValidationPropertyLabels(): array;
```

Comment:
```suggestion
    public function getPropertyLabels(): array;
```

Reply:
Or was the name chosen not to conflict with class method names?

Reply:
Yes, @vjik suggested replacing `getLabels` with `getValidationPropertyLabels`